debug-ui/scripts/setup-market.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  Connection,
+  Keypair,
+  sendAndConfirmTransaction,
+  PublicKey,
+  SystemProgram,
+  TransactionInstruction,
+  Transaction,
+} from '@solana/web3.js';
+import { ManifestClient } from '@cks-systems/manifest-sdk/client';
+import { PROGRAM_ID } from '@cks-systems/manifest-sdk/manifest';
+import { Market } from '@cks-systems/manifest-sdk/market';
+import {
+  airdropSol,
+  getClusterFromConnection,
+} from '@cks-systems/manifest-sdk/utils/solana';
+import { createMint } from '@solana/spl-token';
+import { FIXED_MANIFEST_HEADER_SIZE } from '@cks-systems/manifest-sdk/constants';
+
+async function testCreateMarket(): Promise<void> {
+  const connection: Connection = new Connection('http://127.0.0.1:8899');
+  const payerKeypair: Keypair = Keypair.generate();
+  const marketAddress: PublicKey = await createMarket(connection, payerKeypair);
+
+  const market: Market = await Market.loadFromAddress({
+    connection,
+    address: marketAddress,
+  });
+  market.prettyPrint();
+}
+
+export async function createMarket(
+  connection: Connection,
+  payerKeypair: Keypair,
+): Promise<PublicKey> {
+  const marketKeypair: Keypair = Keypair.generate();
+  console.log(`Cluster is ${await getClusterFromConnection(connection)}`);
+
+  // Get SOL for rent and make airdrop states.
+  await airdropSol(connection, payerKeypair.publicKey);
+  const baseMint: PublicKey = await createMint(
+    connection,
+    payerKeypair,
+    payerKeypair.publicKey,
+    payerKeypair.publicKey,
+    9,
+  );
+  const quoteMint: PublicKey = await createMint(
+    connection,
+    payerKeypair,
+    payerKeypair.publicKey,
+    payerKeypair.publicKey,
+    6,
+  );
+  console.log(`Created baseMint ${baseMint} quoteMint ${quoteMint}`);
+
+  const createAccountIx: TransactionInstruction = SystemProgram.createAccount({
+    fromPubkey: payerKeypair.publicKey,
+    newAccountPubkey: marketKeypair.publicKey,
+    space: FIXED_MANIFEST_HEADER_SIZE,
+    lamports: await connection.getMinimumBalanceForRentExemption(
+      FIXED_MANIFEST_HEADER_SIZE,
+    ),
+    programId: PROGRAM_ID,
+  });
+
+  const createMarketIx = ManifestClient['createMarketIx'](
+    payerKeypair.publicKey,
+    baseMint,
+    quoteMint,
+    marketKeypair.publicKey,
+  );
+
+  const tx: Transaction = new Transaction();
+  tx.add(createAccountIx);
+  tx.add(createMarketIx);
+  const signature = await sendAndConfirmTransaction(
+    connection,
+    tx,
+    [payerKeypair, marketKeypair],
+    {
+      commitment: 'finalized',
+    },
+  );
+  console.log(`Created market at ${marketKeypair.publicKey} in ${signature}`);
+  return marketKeypair.publicKey;
+}
+
+describe('Create Market test', () => {
```

Comment:
Is this file supposed to be a test, not a script?

Reply:
this should be deleted. I copied over from somewhere and was going to use it to setup  state for the UI but never got around to it. rando-bot did well enough for those purposes.